healthcheck/utils.py

```diff
@@ -93,6 +94,28 @@ def extract_reduced_accuracy_lat_long(location):
         loc = Location(location)
         lat = round(float(loc.lat.degrees), 1)
         lng = round(float(loc.lng.degrees), 1)
```

Comment:
The `loc.lng` and `loc.lat` variables have a `sign` property, so you could do something like this:
`lng = round(float(loc.lng.degrees), 1) * int(f"{loc.lng.sign}1")`

Reply:
There's also `.decimal`, which includes the degrees, minutes, seconds, and sign into a single `Decimal` object, which is probably better to use.